src/main/java/appeng/parts/reporting/ConversionMonitorPart.java

```diff
@@ -187,14 +198,12 @@ private void extractItem(Player player, int count) {
             var retrieved = StorageHelper.poweredExtraction(energy, cell, itemKey, count,
                     new PlayerSource(player, this));
             if (retrieved != 0) {
-                ItemStack newItems = itemKey.toStack((int) retrieved);
+                var newItems = itemKey.toStack((int) retrieved);
                 if (!player.getInventory().add(newItems)) {
                     player.drop(newItems, false);
                 }
 
-                if (player.containerMenu != null) {
-                    player.containerMenu.broadcastChanges();
-                }
+                player.containerMenu.broadcastChanges();
```

Comment:
Why? Doesn't this cause a crash if the player is not in a menu?

Reply:
The check was supposedly redundant due to `containerMenu` always being non-null at that point.

Reply:
Why is it always non-null? `extractItem` is called when the player clicks the monitor to take an item, right?